package.json

```diff
@@ -13,8 +13,8 @@
   },
   "scripts": {
     "start": "run-s build:watch",
-    "build": "tsc",
-    "build:watch": "tsc --watch",
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig.cjs.json",
```

Comment:
ideally, we'd use only a single TS project that's a "composite project".

this will allow us to output both versions with build and watch simultaneously.

see here for an example https://github.com/NotionX/react-notion-x/blob/master/tsconfig.json